src/main/java/seedu/uninurse/commons/core/Messages.java

```diff
@@ -11,4 +11,6 @@ public class Messages {
     public static final String MESSAGE_INVALID_TASK_INDEX = "The task index provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
 
+    public static final String MESSAGE_DUPLICATE_TASK = "This task already exists for this patient";
```

Comment:
Perhaps it may be better to standardise where the messages are placed, and what exactly the message is. In `MedicationList` and `ConditionList`, the message for duplicate values is placed in the respective `AddXXCommand` class, and we've standardised the error message to include other details. Do take a look at those classes for reference since they're all quite similar.

Reply:
Similar as above.